Formula/pinot.rb

```diff
@@ -0,0 +1,82 @@
+class Pinot < Formula
+  desc "Realtime distributed OLAP datastore"
+  homepage "https://pinot.apache.org/"
+  url "https://downloads.apache.org/pinot/apache-pinot-0.9.3/apache-pinot-0.9.3-bin.tar.gz"
+  sha256 "c253eb9ce93f11f368498229282846588f478cb6e0359e24167b13e97417c025"
+  license "Apache-2.0"
+  revision 1
+  head "https://github.com/apache/pinot.git", branch: "master"
+
+  depends_on "libtool" => :build
+
+  depends_on "openjdk"
+  depends_on "pinotbase"
+  depends_on "pinotbroker"
+  depends_on "pinotserver"
+  depends_on "zookeeper"
+
+  def default_pinot_env
+    <<~EOS
+      [ -z "$JAVA_OPTS" ] && export JAVA_OPTS="-Xms4G -Xmx8G -XX:+UseG1GC -XX:MaxGCPauseMillis=200 -Xloggc:gc-pinot-controller.log -Dlog4j2.configurationFile=/usr/local/etc/pinot/log4j2.xml"
+    EOS
+  end
+
+  def install
+    data = var/"lib"
+
+    inreplace "conf/pinot-controller.conf",
+      "controller.data.dir=/tmp/pinot/data/controller", "controller.data.dir=#{data}/pinot/data/controller"
```

Comment:
```suggestion
    inreplace "conf/pinot-controller.conf",
              "controller.data.dir=/tmp/pinot/data/controller",
              "controller.data.dir=#{data}/pinot/data/controller"
```
Align the arguments to `inreplace` if you have to split them up. Same below.